types/puppeteer/index.d.ts

```diff
@@ -401,6 +401,8 @@ export interface StyleTagOptions {
   path?: string;
   /** Raw CSS content to be injected into frame. */
   content?: string;
+  /** Script type. Use 'module' in order to load a Javascript ES6 module. */
+  type?: string;
```

Comment:
I'm assuming this should go to `ScriptTagOptions` and not to `StyleTagOptions`? The `type` property isn't available on style-tags according to https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageaddstyletagoptions

Reply:
So sorry. That is my mistake.